lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -130,12 +123,8 @@ negotiate_auto_ethernet:
 
 negotiate_auto_other_interfaces:
   kind: boolean
-  config_get: ~
-  config_set: ~
   default_value: false
-  test_config_result:
-    false: RuntimeError
-    true: RuntimeError
+  default_only: yes
```

Comment:
I think YAML actually supports 'yes' and 'no' and even 'on' and 'off', which is why this didn't fail for me. But I agree this needs to be changed - I'll also add this as a best practice in the README.